engine/classes/Elgg/Notifications/NotificationsService.php

```diff
@@ -47,13 +53,23 @@ class NotificationsService {
 	 * @param \Elgg\Queue\Queue                        $queue         Queue
 	 * @param \Elgg\PluginHooksService                 $hooks         Plugin hook service
 	 * @param \ElggSession                             $session       Session service
+	 * @param \Elgg\I18n\Translator                    $translator    Translator
+	 * @param \Elgg\Database\EntityTable               $entities      Entity table
 	 */
-	public function __construct(\Elgg\Notifications\SubscriptionsService $subscriptions,
-			\Elgg\Queue\Queue $queue, \Elgg\PluginHooksService $hooks, \ElggSession $session) {
+	public function __construct(
+			\Elgg\Notifications\SubscriptionsService $subscriptions,
+			\Elgg\Queue\Queue $queue,
+			\Elgg\PluginHooksService $hooks,
+			\ElggSession $session,
+			\Elgg\I18n\Translator $translator,
+			\Elgg\Database\EntityTable $entities) {
```

Comment:
Long term makes some sense to look at which services always appear together and consider merging them, but shorter term probably not worth worrying about.